testsuite/features/support/env.rb

```diff
@@ -368,6 +384,10 @@ def enable_assertions
   skip_this_scenario if $rocky9_minion || $rocky_ssh_minion
 end
 
+Before('@skip_for_sle_micro') do
+  skip_this_scenario if $slemicro53_minion || $slemicro53_ssh_minion || $slemicro52_minion || $slemicro52_ssh_minion
```

Comment:
That will not do what you think it will :)
This will skip all the scenarios having this tag if exist a SLE Micro deployed in your environment.

Instead you need:

```suggestion
Before('@skip_for_sle_micro') do
  skip_this_scenario if scenario.location.file.include? 'slemicro'
end
```